BEP3.md

```diff
@@ -66,16 +66,16 @@ APS is designed to support peg token from any EVM based blockchain or any one wi
 3. Client gets guarantee on their fund safety. They don’t need to worry too much about the availability of the Deputy process, or any cheat from issuers or swap service providers.
 4. It is possible for one set of infrastructure to handle swap for multiple tokens.
 
-#### Client Swap Tokens from Ethereum to Binance Chain
+#### Client Swap Tokens from Ethereum to BNB Beacon Chain
 
 1. Client calls APS contract, with the hash of a secret, X tokens and a time span T parameters, to express his/her interest to swap X tokens. If the parameters are good (e.g. T>MinLockTime, enough tokens to swap), the call transaction is recorded on the blockchain.
-2. Deputy process monitors the events of the APS contract on Ethereum. If it detects the client’s call and verify good, it will sign and broadcast the HTLT transaction on Binance Chain. This will lock X (or more as bonus) number of pegged tokens on Binance Chain. Please note the time span used in this HTLT transaction should be smaller enough than T in Client’s APS call.
-3. Client or Client tooling monitors any transactions onto CB, if it is from OB and has the proper hash, Client or Client tooling should broadcast CHLT transaction to claim his/her requested Pegged tokens by disclosing the random number generating the hash. Binance Chain will verify the random number, if it matches the hash, it will release the locked Pegged tokens to Client address CB.
+2. Deputy process monitors the events of the APS contract on Ethereum. If it detects the client’s call and verify good, it will sign and broadcast the HTLT transaction on BNB Beacon Chain. This will lock X (or more as bonus) number of pegged tokens on BNB Beacon Chain. Please note the time span used in this HTLT transaction should be smaller enough than T in Client’s APS call.
```

Comment:
**Grammar Error**

_Corrected_
The Deputy process monitors the events of the APS contract on Ethereum. If it detects a Client's call and verifies it is valid, it will sign and broadcast the HTLT transaction on BNB Beacon Chain. This will lock X (or more as a bonus) number of pegged tokens on BNB Beacon Chain. Please note that the time span used in this HTLT transaction should be shorter than T in the Client's APS call.

Reply:
updated